runbotics/runbotics-orchestrator-ui/src/src-app/Actions/excel.actions.ts

```diff
@@ -136,6 +136,39 @@ const getExcelActions: () => Record<string, IBpmnAction> = () => ({
             formData: {},
         },
     },
+    'excel.clearCells': {
+        id: 'excel.clearCells',
+        label: translate('Process.Details.Modeler.Actions.Excel.ClearCells.Label'),
+        script: 'excel.clearCells',
+        runner: Runner.DESKTOP_SCRIPT,
+        system: ActionSystem.WINDOWS,
+        form: {
+            schema: {
+                type: 'object',
+                properties: {
+                    input: {
+                        title: translate('Process.Details.Modeler.Actions.Common.Input'),
+                        type: 'object',
+                        properties: {
+                            targetCells: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.ClearCells.TargetCells'),
+                                type: 'string'
+                            },
+                            worksheet: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.Worksheet'),
+                                type: 'string',
+                            }
+                        },
+                        required: ['targetCells'],
+                    },
+                },
+            },
+            uiSchema: {
+                'ui:order': ['input'],
+            },
+            formData: {},
```

Comment:
If you checked that there won't be any typeErrors then I think it is ok to leave formData as empty object.

Reply:
why would it be?
I suppose it should be 
            formData: {
                output: {
                    variableName: undefined,
                }
            },
?